tests/virtualization/universal/patch_and_reboot.pm

```diff
@@ -26,6 +26,8 @@ use qam;
 sub run {
     my $self       = shift;
     my $kernel_log = shift // '/tmp/virt_kernel.txt';
+    # Use serial terminal, unless defined otherwise. The unless will go away once we are certain this is stable
+    $self->select_serial_terminal unless get_var('_VIRT_SERIAL_TERMINAL', 1) == 0;
```

Comment:
I'd add those new variables to `variables.md` file.

Reply:
Also for a temporary switch? The comment says "The unless will go away once we are certain this is stable".

Reply:
This should never be used and is also only meant for us internally.